.github/workflows/pr-images.yaml

```
@@ -0,0 +1,93 @@
+      - name: Install Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: .build/go.mod
+          check-latest: true
+          cache-dependency-path: |
+            .build/go.sum
+
+      - name: Execute
+        id: refs
+        working-directory: .build
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:8:8: Double quote to prevent globbing and word splitting

(shellcheck)